deposits.md

```diff
@@ -23,11 +23,12 @@ fields (rlp encoded in the order they appear here):
 
 [EIP-2718]: https://eips.ethereum.org/EIPS/eip-2718
 
-- `address from`
-- `address to`
-- `uint256 value`
-- `bytes data`
-- `uint256 gasLimit`
+- `address from`: The address of the sender account.
+- `address to`: The address of the recipient account.
+- `uint256 depositValue`: The ETH value to mint on L2.
+- `uint256 sendValue`: The ETH value to send to the recipient account.
```

Comment:
Another option would be something like this, which might be cleaner because it adds a new field alongside the existing `value` field in other transaction types, rather than replacing it with 2 new fields.

```suggestion
- `uint256 deposited`: The ETH value to mint on L2.
- `uint256 value`: The ETH value to send to the recipient account.
```